pkg/ccl/crosscluster/logical/create_logical_replication_stmt.go

```diff
@@ -178,6 +178,22 @@ func createLogicalReplicationStreamPlanHook(
 			}
 		}
 
+		// Commit the planner txn because several operations below may take several
+		// seconds, which we would like to conduct outside the scope of the planner
+		// txn to prevent txn refresh errors.
+		if err := p.Txn().Commit(ctx); err != nil {
```

Comment:
did we already have a check above this that asserted the planner txn was implicit ?

Reply:
oh lol, just found something.

Reply:
fixed.

Reply:
Wait, just to confirm, backup *does* check this, right?

Reply:
on non detached backups:
https://github.com/msbutler/cockroach/blob/butler-different-default-values/pkg/ccl/backupccl/backup_planning.go#L559

Reply:
Ah, good good. Had me worried there for a minute. Carry on.